src/fsharp/service/service.fs

```diff
@@ -846,6 +846,12 @@ type BackgroundCompiler(legacyReferenceResolver, projectCacheSize, keepAssemblyC
                 if startBackgroundCompileIfAlreadySeen then 
                    bc.CheckProjectInBackground(options, userOpName + ".StartBackgroundCompile"))
 
+    member bc.ClearCache(options : FSharpProjectOptions list, userOpName) =
```

Comment:
if this is `seq<FSharpProjectOptions>` then you won't need to convert the dictionary into a list on the VS side

Reply:
I think it makes sense, considering FSharpProjectOptions is considered immutable.

Reply:
since it just iterates through the collection, seq should be sufficient typing.